src/composables/transferSendForm.ts

```diff
@@ -149,7 +149,7 @@ export function useTransferSendForm({
       formModel.value.address = parsedScanResult.tokenContract;
       formModel.value.amount = toShiftedBigNumber(
         parsedScanResult.amount,
-        -formModel.value.selectedAsset?.decimals!,
+        formModel.value.selectedAsset?.decimals ? -formModel.value.selectedAsset.decimals : 0,
```

Comment:
What do you think about this one?
```suggestion
        -(formModel.value.selectedAsset?.decimals || -0),
```